core-cube/src/test/java/org/apache/kylin/cube/cuboid/CuboidSchedulerTest.java

```diff
@@ -118,8 +118,13 @@ private void testSpanningAndGetParent(CuboidScheduler scheduler, CubeDesc cube,
         }
 
         long[] spanningsArray = Longs.toArray(totalSpanning);
-        Arrays.sort(spanningsArray);
-        Arrays.sort(expectChildren);
+
+        /*
+            According to issue : https://issues.apache.org/jira/browse/KYLIN-3631
```

Comment:
This comment is not needed. Can you remove it in all the places?

Reply:
Sure